### PR TITLE
Rpm install instructions

### PR DESCRIPTION
**Pull request by qrkourier**

### CODE REVIEW

**Comment by qrkourier** [comment]
Branched from pending PR in https://github.com/openziti/ziti-doc/pull/138

File: docfx_project/ziti/clients/tunneler-overview.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-A tunneler is purpose-built software designed to connect applications which are not Ziti-aware to the Ziti Network.
-Ziti provides tunnelers for each major operating system.
+
+An OpenZiti Tunneler is purpose-built software designed to connect applications which are not Ziti-aware to the OpenZiti Network. Ziti Tunnellers allow pre-existing, unmodified TCP/IP applications to access or provide services on a secure OpenZiti Network. An OpenZiti Tunneler cannot secure the communication between itself and the TCP/IP client or server application.
```
Comment (dovholuknf) [inline docfx_project/ziti/clients/tunneler-overview.md]: I'm sorry but everything after the lead is redundant information to me. Not to mention that it doesn't mention UDP... it's not just TCP. A tunneler bridges/connects/enables non-ziti-embedded software to a ziti network. all the rest is just extra words. This section doesn't need these extra words

"Ziti Tunnellers allow pre-existing, unmodified TCP/IP applications to access or provide services on a secure OpenZiti Network. An OpenZiti Tunneler cannot secure the communication between itself and the TCP/IP client or server application"

File: docfx_project/ziti/clients/tunneler.md [hunk 2]
```diff
@@ -6,11 +6,10 @@ Each tunneller operates similarly. The goal is to have the tunneler intercecpt t
...
 tunneler operate in: seamless and proxy. A seamless tunneler will transparently intercept traffic via IPv4 address or
 DNS whereas a tunneler in proxy mode works as a proxy. Seamless mode is transparent to existing services and
-applications. Proxy mode is not as transparent at all. It requires applications to send traffic to the localhost proxy
-specifically. This means when running in proxy mode - it does not do any intercepting at all.
+applications. Proxy mode is not transparent at all. It requires applications to send traffic to the localhost proxy
+specifically. This means when running in proxy mode it does not do any intercepting at all.
 
-Here you can learn about the
-ways each tunneler operates and see the similarities as well as any differences.
+Ziti provides tunnelers for each major operating system.
```
Comment (dovholuknf) [inline docfx_project/ziti/clients/tunneler.md]: Ziti ->OpenZiti or better, "The OpenZiti project provides tunnelers" 

And as an aside... I need to figure out how to find the authoritative spelling we should follow... Two l's or one???

Word's spellchecker still tells me that "tunneler" is wrong while "tunneller" is correct.... Maybe we need twitter for this answer. I tried looking it up on Oxford Dictionary but failed. lolol

image

File: docfx_project/ziti/clients/linux.md [hunk 3]
```diff
@@ -8,7 +8,51 @@ The purpose of the tunneller is to configure host access. This means all users a
 
 ### Installation and Upgrade
 
-[The latest release](https://github.com/openziti/ziti-tunnel-sdk-c/releases/latest/) of `ziti-edge-tunnel` is distributed as a binary executable from GitHub. The upgrade procedure is identical to the installation procedure.
+#### RPM
+
+To install the x86_64 RPM you first create a repo file like `/etc/yum.repos.d/openziti.repo` resembling the examples below. Then, you run `yum update` to refresh your repodata cache so you can run `yum install ziti-edge-tunnel` to install the RPM. To upgrade you run `yum update ziti-edge-tunnel`.
```
Comment (dovholuknf) [inline docfx_project/ziti/clients/linux.md]: Does it only 'resemble' the examples below? Can we not be authoritative and state exactly like the example below? I would rather prescribe to people 'how to' and be authoritative. Is that not possible?

I try to not use "you" in this doc - instead reword it to say "Then run" or "once in place, issue this command". I think this could be bullet points too?

File: docfx_project/ziti/clients/linux.md [hunk 4]
```diff
@@ -8,7 +8,51 @@ The purpose of the tunneller is to configure host access. This means all users a
...
+
+The final setup step is to install an enroll token JWT file or identity config JSON file in `/opt/openziti/etc/identities` before you run `systemd start ziti-edge-tunnel.service`. The service needs to be restarted if the contents of the identities directory change.
+
+##### RHEL, CentOS, and Rocky Linux
+
+```ini
+[OpenZiti]
+name=OpenZiti
+baseurl=https://netfoundry.jfrog.io/artifactory/zitipax-openziti-rpm-stable/redhat$releasever/$basearch
```
Comment (dovholuknf) [inline docfx_project/ziti/clients/linux.md:24]: there's no chance to proxy netfoundry.jfrog.io to openziti.org is there? I feel like this is one of those "better to get it right the first time" things?

File: docfx_project/ziti/clients/linux.md [hunk 5]
```diff
@@ -8,7 +8,51 @@ The purpose of the tunneller is to configure host access. This means all users a
...
+baseurl=https://netfoundry.jfrog.io/artifactory/zitipax-openziti-rpm-stable/redhat7/$basearch
+enabled=1
+gpgcheck=0
+gpgkey=https://netfoundry.jfrog.io/artifactory/zitipax-openziti-rpm-stable/redhat7/$basearch/repodata/repomd.xml.key
+repo_gpgcheck=1
+```
+
+#### Binary
+
+[The latest binary release](https://github.com/openziti/ziti-tunnel-sdk-c/releases/latest/) of `ziti-edge-tunnel` is distributed as an executable from GitHub for amd64, arm, arm64 architectures. The upgrade procedure is identical to the installation procedure.
```
Comment (dovholuknf) [inline docfx_project/ziti/clients/linux.md]: i'd just leave "from GitHub" out of here. If we ever move it, I could see someone fixing the link, but not this text...

File: docfx_project/ziti/clients/tunneler-overview.md (hunk 1 above)
Comment (qrkourier) [reply docfx_project/ziti/clients/tunneler-overview.md]: Will send commit in https://github.com/openziti/ziti-doc/pull/138

File: docfx_project/ziti/clients/tunneler.md (hunk 2 above)
Comment (qrkourier) [reply docfx_project/ziti/clients/tunneler.md]: Will send commit in https://github.com/openziti/ziti-doc/pull/138

File: docfx_project/ziti/clients/linux.md (hunk 4 above)
Comment (qrkourier) [reply docfx_project/ziti/clients/linux.md:24]: I'm exploring this possibility.

**Review by dovholuknf — APPROVED** [review]
